app/controllers/miq_ae_tools_controller.rb

```diff
@@ -335,14 +335,6 @@ def reset_datastore
 
   private ###########################
 
-  def ae_logger_contents(full: false)
-    return unless $miq_ae_logger
-
-    contents = full ? $miq_ae_logger.contents(nil, nil) : $miq_ae_logger.contents
-    contents = nil if contents.start_with?("# Logfile created on") && !contents.include?("\n")
```

Comment:
This is a pointless check, IMO, so I'm removing it.

Reply:
yeah, WAT.